lib/files.js

```diff
@@ -253,6 +263,7 @@ function getComponentPackage(name) {
 }
 
 exports.getYaml = control.memoize(getYaml);
+exports.getYamlWithEnv = control.memoize(getYamlWithEnv);
```

Comment:
This function seems like you just copy and pasted the function above it.  Did you just copy and paste the function above it?

Why do you want to memoize this function?  Do you see it being called repeatedly in short periods of time?

I think more thought might be needed here.